Selenium/exercise06.rb

```diff
@@ -0,0 +1,17 @@
+require 'selenium-webdriver'
+
+Selenium::WebDriver::Chrome::Service.driver_path="C:/Ruby32-x64/Drivers/chromedriver.exe"
+driver = Selenium::WebDriver.for :chrome
+
+driver.get 'https://letcode.in/forms'
+
+# main = driver.find_elements(:class, 'control')
+dropdown = driver.find_element(:xpath, '/html/body/app-root/app-forms/section[1]/div/div/div[1]/div/div/form/div[2]/div[2]/div/div/div/select')
+# values = main.find_element(:tag_name, 'select')
+values = dropdown.find_elements(:tag_name, 'option')
```

Comment:
A better approach is to use

`dropdown = Selenium::WebDriver::Support::Select.new(dropdown_xpath)`

You should make use of the functions mentioned here: https://www.selenium.dev/selenium/docs/api/rb/Selenium/WebDriver/Support/Select.html